template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/LoginWith2fa.cshtml.cs

```diff
@@ -77,17 +77,17 @@ public class InputModel
 
             if (result.Succeeded)
             {
-                _logger.LogInformation("{UserName} logged in with 2fa.", user.UserName);
+                _logger.LogInformation("User with id '{UserId}' logged in with 2fa.", user.Id);
```

Comment:
If I already have the user, why do I need to back through the manager to get the ID?